library/src/webapp/editor/ckextraplugins/common/css/dialog.css

```diff
@@ -0,0 +1,27 @@
+/* search results paging styles */
```

Comment:
It's so I don't have to repeat myself in each widget's dialog.css.  The Youtube, Vimeo, and CreativeCommons widgets all now use the same paging styling and so share some css classes.